src/redux/books/books.js

```diff
@@ -1,59 +1,96 @@
-import uuid from 'react-uuid';
-
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ewYk4YjZVq8pYtMAfktj/books';
 const ADD = 'BOOK_ADDED';
 const DEL = 'BOOK_REMOVED';
+const READ = 'BOOKS_RETRIEVED';
 
 // Action creators
-export const addBook = ({ title, author }) => ({
+export const addBook = ({
+  // eslint-disable-next-line camelcase
+  item_id, title, author, category,
+}) => ({
   type: ADD,
-  id: uuid(),
+  item_id,
   title,
   author,
+  category,
 });
 
-export const removeBook = (id) => ({
+// eslint-disable-next-line camelcase
+export const removeBook = (item_id) => ({
   type: DEL,
-  id,
+  item_id,
+});
+
+export const getActionData = ({
+  // eslint-disable-next-line camelcase
```

Comment:
- [ ] As requested by the previous reviewer, please fix the Linters errors rather than disabling them.